docs/CHANGELOG.rst

```diff
@@ -7,6 +7,17 @@ file.
 This project adheres to `Semantic Versioning <http://semver.org/>`_.
 
 ==================
+1.2.0 - 2017-08-08
+==================
+
+Changed
+-------
+- Unexpected errors in ``re-*`` functions are handled and error with
```

Comment:
I would change the sentence to imperative mode, something like:
Handle unexpected errors in ``re-*`` functions and print an error with a descriptive message.